test/extensions/tracers/opentelemetry/opentelemetry_tracer_impl_test.cc

```diff
@@ -559,6 +564,7 @@ TEST_F(OpenTelemetryDriverTest, ExportOTLPSpanWithAttributes) {
   span->setTag("", "empty_tag_value");
   // Overwrite a tag.
   span->setTag("first_tag_name", "first_tag_new_value");
+  span->setTag("http.status_code", "500");
```

Comment:
Consider adding this in its own explicit test (avoiding a test covering multiple use-cases).